apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -18,6 +19,7 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     // bytes32 public constant CREATE_VOTES_ROLE = keccak256("CREATE_VOTES_ROLE");
     bytes32 public constant CREATE_VOTES_ROLE = 0xe7dcd7275292e064d090fbc5f3bd7995be23b502c1fed5cd94cfddbbdcd32bbc;
 
+    // Note: maybe we should just use CHANGE_* as a prefix?
```

Comment:
Minor nitpick, but we call the functions "change*" so might as well name the roles similarly :)

Reply:
Addressed in d131b514f74993aa434e2ffc701dc74a030a9b58